polyply/src/minimizer.py

```diff
@@ -42,7 +42,7 @@ def compute_bond(params, coords):
     float
     """
     dist = np.linalg.norm(coords[0] - coords[1])
-    return 1000 *(dist - float(params[1]))**2.0
+    return 10000 *(dist - float(params[1]))**2.0
```

Comment:
Why? Also, sounds like it should be an argument (with a default)

Reply:
actually this is carefully chosen to be 1000, 10000 is a leftover from some testing. The idea is that anlges are for example values from 0-300 while bonds tend to be from 0.1 to 0.5 by multiplying with 1000 they go from 100-5000 so there is more emphasis on bonds without overpowering angle values in the optimization. But we can make it an argument with default

Reply:
also here the new system builder will change the parsing of arguments to these modules and all this is going to be exposed to CLI